specification/search/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the SearchManagementClient API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-11
```

Comment:
why you change the apiversion to a older preview one? 2023-11 was released last month

Reply:
> why you change the apiversion to a older preview one? 2023-11 was released last month

Because @HeidiSteen want that older preview version shown in the documentation, stage version tag can be generated by default, we set default tag in basic information as that preview version to achieve her request.

Reply:
@Kazrael2119, it looks like I accidentally have 2 PRs for this edit.  In PR 26513, I noted the following;

The new stable version should still be there. My understanding is that all stable versions show up in the docs automatically, and if some of those versions aren't needed, they're suppressed during the doc autogen process. This fix is an attempt to restore preview APIs docs, which we used to have, but disappeared when the platform team adopted the new filtering mechanism. I was told that if I list the preview API package in the readme. I'll get the preview API docs back. That's what I'm trying to accomplish here.

I'm going to close the duplicate PR (https://github.com/Azure/azure-rest-api-specs/pull/26513) and keep this one.  Sorry for the churn.